js/find-the-culprit.js

```diff
@@ -354,7 +354,9 @@ async function deactivationStep(chosenModules = []) {
     await game.settings.set(moduleName, "modules", currSettings);
   }
 
-  game.settings.set("core", ModuleManagement.CONFIG_SETTING, original);
+  game.settings.set("core", ModuleManagement.CONFIG_SETTING, original).then(r => {
```

Comment:
This is inside an asynchronous function, so it makes more sense to `await` this line and then put the reload on the next line without using `then`. This is more in line with modern JavaScript best practices.